src/Microsoft.Owin.Security.Authorization/Infrastructure/AppBuilderExtensions.cs

```diff
@@ -6,7 +6,7 @@ namespace Microsoft.Owin.Security.Authorization.Infrastructure
 {
     public static class AppBuilderExtensions
     {
-        public static IAppBuilder UseAuthorization(this IAppBuilder app, AuthorizationOptions options, AuthorizationDependenciesProvider dependenciesProvider = null)
+        public static IAppBuilder UseAuthorization(this IAppBuilder app, AuthorizationOptions options, IAuthorizationDependenciesProvider dependenciesProvider = null, IAuthorizationHandler[] handlers = null)
```

Comment:
The project engineering guidelines state that we should shy away from default parameters.  For an explanation why, go here: https://msdn.microsoft.com/library/ms182135.aspx